test/common.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+global.chai = require('chai').use(require('sinon-chai'));
+global.should = chai.should();
+global.sinon = require('sinon');
+require('sinomocha')();
+require('moment-business');
```

Comment:
I don't think this should be a hard dependency - rather only required in apps that use it